react/react1/week3/to-do/src/components/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import "./to-dos.css";
+import { React, useEffect, useState } from "react";
+const ToDoHeader = () => {
+  const [timeSpent, setTimeSpent] = useState(0);
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimeSpent(timeSpent + 1);
+    }, 1000);
+    return () => {
+      clearInterval(interval);
+    };
+  });
+  return (
+    <div>
+      <h3 className="header">
+        You have sepnt <em style={{ color: "green" }}>{timeSpent}</em> seconds
```

Comment:
It is advised to avoid inline styles by all means as they are difficult to track, especially when you have style files and other styling in the style files.

Reply:
👍🏻sure will keep in mind